### PR TITLE
Extract resource header #LMR-856

### PR DESCRIPTION
**Pull request by kubedo8**

### CODE REVIEW

File: src/app/core/store/projects/projects.effects.ts [hunk 1]
```diff
@@ -143,10 +146,26 @@ export class ProjectsEffects {
...
+      const paramMap = RouteFinder.getFirstChildRouteWithParams(this.router.routerState.root.snapshot).paramMap;
+      const projCodeInRoute = paramMap.get('projectCode');
+
+      if (projCodeInRoute && projCodeInRoute === oldProject.code && project.code !== oldProject.code) {
+        const paths = this.router.routerState.snapshot.url.split('/').filter(path => path);
+        const index = paths.indexOf(oldProject.code, 3);
+        if (index !== -1) {
+          paths[index] = project.code;
+          actions.push(new RouterAction.Go({path: paths}));
+          // TODO extract as
```
Comment (livthomas) [inline src/app/core/store/projects/projects.effects.ts]: As what? :)

File: src/app/workspace/project/project-settings.component.ts [hunk 2]
```diff
@@ -17,139 +17,70 @@
...
+    this.subscription.unsubscribe();
 
-  public updateProject(): void {
-    this.projectService.editProject(this.organizationCode, this.projectCode, this.project).subscribe(
-      success => null,
-      error => {
-        const message = this.i18n({id: 'project.update.fail', value: 'Failed to update project.'});
-        this.notificationService.error(message);
-      });
+    this.store.dispatch(new CollectionsAction.Clear());
```
Comment (livthomas) [inline src/app/workspace/project/project-settings.component.ts]: Why are collections cleared when you leave project settings page?

Comment (kubedo8) [reply src/app/workspace/project/project-settings.component.ts]: I will discuss this on tomorrow meeting.

File: src/app/core/store/projects/projects.effects.ts (hunk 1 above)
Comment (kubedo8) [reply src/app/core/store/projects/projects.effects.ts]: This TODO comment is also in `organizations.effects.ts` and I don't know what it means. So I decided to leave it here too.

**Pull request merged**